release notes/v0.26.0.md

```diff
@@ -0,0 +1,120 @@
+
+### `console` is now available in the init context (#982):
+
+For a long time, this wasn't supported which made debugging things outside of VU code much harder, but now it's here.
+
+There is currently known bug is that calls to console in the init context will always write to the stdout regardless of using `--console-output`.
+
+### HTTP: request body decompression with multiple layered algorithms (#1125)
+
+In v0.25.0 compressing bodies was added and it had support for multiple layered algorithms. Now this is also true for decompressing bodies when k6 gets them as responses.
+
+
+### Summary: add an optional count column (#1143)
+
+The `--summary-trend-stats` now also recognizes `count` as a valid column and will output the count of samples in all Trend metrics.
+
+### Docker compose refactor (#1183)
+
+The example docker-compose that enabled easy running of InfluxDB+Grafana+k6 was refactored and all the images were updated to use the latest stable versions.
+
+Thanks, @KajdeMunter!
+
+Also the k6 Dockerfile alpine version was bumped to 3.10. Thanks @dmitrytokarev!
+
+## Bugs fixed!
+* Cloud: Stop sending metrics to the cloud output when the cloud returns that you have reached the limit (#1130)
+* JS: Fail a `check` if an uncaught error is thrown inside of it (#1137)
+* HTTP: Replace user credentials with `*` when emitting http metrics (#1132)
+* WS: Many fixes:
+  - return error instead of panicking if error happens during the making of the websocket connection (#1127)
+  - calling the `error` handler on an error when closing the websocket, instead of calling with a null (#1118)
+  - correctly handle server initiated close (#1186)
+* JSON: Better error messages when parsing JSON fails. Now telling you at which line and row the
+  error is instead of just the offset. Thanks, @openmohan (#905)
+* HTTP: Use Request's `GetBody` in order to be able to get the body multiple times for a single
+  request as needed in 308 redirects of posts and if the server sends GOAWAY with no error (#1093)
+* JS: Don't export internal go struct fields of script options (#1151)
+* JS: Ignore `minIterationDuration` for `setup` and `teardown` (#1175)
+* UX: Better timeout messages for `setup` and `teardown` timeouts including a hint on how to fix it (#1173)
+* HTTP: Return error on any request that returns 101 status code as k6 currently doesn't support any protocol upgrade behaviour (#1172)
+* HTTP: Correctly capture TCP reset by peer and broken pipe errors and give them the appropriate error_code (#1164)
+* UX: Don't interpret not `K6_` prefixed env variables as k6 configuration, most notable `DURATION` and `ITERATIONS` (#1215)
+* JS/html: When using `Selection.map` was not wrapping the nodes it was outputting which lead to wrongly using the internal `Goquery.Selection` instead of k6's `Selection`. Thanks to @MMartyn for reporting this! (#1198)
```

Comment:
```suggestion
* JS/html: `Selection.map` was not wrapping the nodes it was outputting, which lead to wrongly using the internal `Goquery.Selection` instead of k6's `Selection`. Thanks to @MMartyn for reporting this! (#1198)
```